frontend/src/lib/components/CopyToClipboardButton.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import { Duration, delay } from '$lib/util/time';
+  import IconButton from './IconButton.svelte';
+  import t from '$lib/i18n';
+
+  var copyingToClipboard = false;
+  var copiedToClipboard = false;
+
+  export let getTextToCopy: () => string;
```

Comment:
Especially in Svelte where things are pretty much always reactive 🤓

Reply:
I dunno, it seemed like there was no need to have it be a reactive parameter when we only really need the text at the moment the button is clicked. But as I look at the code again, I realize there's no gain from that: the Svelte compiler isn't going to be changing the DOM when the `textToCopy` attribute changes, because the only place it's used inside the component is in JS code. So yeah, I've switched it to be a plain string prop.